lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -33,6 +33,7 @@ check_button_disabled:
   element:
     selector:
       xpath: //button[contains(., '<button_text>') and @disabled]
+    timeout: 60
```

Comment:
Why do we add `timeout: 60`? By mistake?

Reply:
This is to fix OCP-21034, add waiting time to avoiding page loading not finished.

Reply:
Then it maybe good if we add rules to make sure page is loaded rather than hard wait, let's talk if that's possible.